app/src/main/java/org/mozilla/focus/utils/Settings.kt

```diff
@@ -370,6 +372,21 @@ class Settings(
         false
     )
 
+    // Store how many tabs were opened until now, but the max value wich can be stored is 3
+    // since this values is used to decide if we should display the erase cfr
+    var numberOfTabsOpened: Int
+        get() = preferences.getInt(
+            context.getString(R.string.number_of_opened_tabs),
+            0
+        )
+        set(value) {
+            if (value <= ERASE_CFR_LIMIT + 1) {
```

Comment:
This contradicts the property description - "the max value which can be stored is `3`".
Is it okay to also store `4`?